Game.py

```diff
@@ -0,0 +1,50 @@
+import Board
+import BoardState
+
+class Game:
+    def __init__(self, player1, player2):
+        self._players = [player1, player2]
+        self._currentBoard = Board()  #### Game.cs 20줄 대신 클래스 인스턴스 변수로 설정.
+
+    def StartLoop(self):
+
+        currentState = BoardState()
+        currentPlayer = 0
+        self.PrintState()
+
+        while True:
+            while True:
+                move = self._players[currentPlayer].NextMove(self._currentBoard)
+                if not self._currentBoard.IsValidMove(move):
```

Comment:
이 부분은 굳이 이렇게 쓰기 보다는
```suggestion
if self._currentBoard.IsValidMove(move):
    break
```
이렇게 쓰는 게 낫지 않을까요? 어차피 continue 안 써줘도 기본적으로 루프는 돌아가니까요